recipe/build_pkg.sh

```diff
@@ -3,7 +3,7 @@
 set -exuo pipefail
 
 # install the whl making sure to use host pip/python if cross-compiling
-${PYTHON} -m pip install --no-deps $SRC_DIR/tensorflow_pkg/*.whl
+${PYTHON} -m pip install --no-deps --no-build-isolation $SRC_DIR/tensorflow_pkg/*.whl
```

Comment:
`--no-build-isolation` is technically redundant here since wheels are not built.

Reply:
Agreed but that made the linter happy and is not harmful.